src/domain/synthetics/leaderboards/useAccountOpenPositions.ts

```diff
@@ -0,0 +1,212 @@
+import useSWR from "swr";
+import { BigNumber, utils } from "ethers";
+import { getAddress } from "ethers/lib/utils";
+import { queryAccountOpenPositions } from "./queries"
+import { arbitrumGoerliLeaderboardsClient as graph } from "lib/subgraph/clients";
+import { AccountOpenPositionJson, AccountOpenPosition } from "./types";
+import { ContractMarketPrices, getContractMarketPrices, useMarketsInfo } from "../markets";
+import { convertToUsd } from "../tokens";
+// import { useChainId } from "lib/chains";
+import { usePositionsInfo } from "./usePositionsInfo";
+import { PositionsInfoData, getPositionKey } from "../positions";
+import { useEffect, useState } from "react";
+import { expandDecimals } from "lib/numbers";
+import { USD_DECIMALS } from "lib/legacy";
+import { AVALANCHE } from "config/chains";
+// import { useEnsBatchLookup } from "./useEnsBatchLookup";
+
+const fetchAccountOpenPositionsPage = async (
+  first: number,
+  skip: number,
+  orderBy: string = "sizeInUsd",
+  orderDirection: "asc" | "desc" = "desc",
+): Promise<Array<AccountOpenPositionJson>> => {
+  const res = await graph.query<{ accountOpenPositions: Array<AccountOpenPositionJson> }>({
```

Comment:
i think we usually use `AccountOpenPositionJson[]` instead of `Array<AccountOpenPositionJson>`

Reply:
just to confirm, does it load positions for specific account(s)? at first glance it loads all positions. i think using "account" in names here could be a bit confusing. for example we there are helpers in Reader contract like `getAccountPositions` that returns positions for account.